Failed to fork child process

I have tried running your script on many different machines all with Ubuntu 16.04 LTS installed. No matter what I do, it always fails to start. Same exact error on all machines.

Build always fails right here
[build CXX] vcl/headless/svptext.cxx
/opt/libreoffice/vcl/null/printerinfomanager.cxx: In constructor â€˜psp::PrinterInfoManager::PrinterInfoManager(psp::PrinterInfoManager::Type)â€™:
/opt/libreoffice/vcl/null/printerinfomanager.cxx:47:33: error: call of overloaded â€˜unique_ptr(NULL)â€™ is ambiguous
     m_aSystemDefaultPaper( "A4" )
                                 ^
In file included from /usr/include/c++/5/memory:81:0,
                 from /opt/libreoffice/include/i18nlangtag/languagetag.hxx:19,
                 from /opt/libreoffice/include/tools/resmgr.hxx:23,
                 from /opt/libreoffice/include/tools/rc.hxx:23,
                 from /opt/libreoffice/include/vcl/outdev.hxx:26,
                 from /opt/libreoffice/include/vcl/window.hxx:25,
                 from /opt/libreoffice/include/vcl/vclevent.hxx:26,
                 from /opt/libreoffice/include/vcl/helper.hxx:26,
                 from /opt/libreoffice/include/vcl/printerinfomanager.hxx:27,
                 from /opt/libreoffice/vcl/null/printerinfomanager.cxx:20:
/usr/include/c++/5/bits/unique_ptr.h:356:7: note: candidate: std::unique_ptr<_Tp, _Dp>::unique_ptr(const std::unique_ptr<_Tp, _Dp>&) [with _Tp = psp::SystemQueueInfo; _Dp = std::default_delete<psp::SystemQueueInfo>] <deleted>
       unique_ptr(const unique_ptr&) = delete;
       ^
In file included from /usr/include/c++/5/memory:81:0,
                 from /opt/libreoffice/include/i18nlangtag/languagetag.hxx:19,
                 from /opt/libreoffice/include/tools/resmgr.hxx:23,
                 from /opt/libreoffice/include/tools/rc.hxx:23,
                 from /opt/libreoffice/include/vcl/outdev.hxx:26,
                 from /opt/libreoffice/include/vcl/window.hxx:25,
                 from /opt/libreoffice/include/vcl/vclevent.hxx:26,
                 from /opt/libreoffice/include/vcl/helper.hxx:26,
                 from /opt/libreoffice/include/vcl/printerinfomanager.hxx:27,
                 from /opt/libreoffice/vcl/null/printerinfomanager.cxx:20:
/usr/include/c++/5/bits/unique_ptr.h:205:7: note: candidate: std::unique_ptr<_Tp, _Dp>::unique_ptr(std::unique_ptr<_Tp, _Dp>&&) [with _Tp = psp::SystemQueueInfo; _Dp = std::default_delete<psp::SystemQueueInfo>]
       unique_ptr(unique_ptr&& __u) noexcept
       ^
/usr/include/c++/5/bits/unique_ptr.h:200:17: note: candidate: constexpr std::unique_ptr<_Tp, _Dp>::unique_ptr(std::nullptr_t) [with _Tp = psp::SystemQueueInfo; _Dp = std::default_delete<psp::SystemQueueInfo>; std::nullptr_t = std::nullptr_t]
       constexpr unique_ptr(nullptr_t) noexcept : unique_ptr() { }
                 ^
In file included from /usr/include/c++/5/memory:81:0,
                 from /opt/libreoffice/include/i18nlangtag/languagetag.hxx:19,
                 from /opt/libreoffice/include/tools/resmgr.hxx:23,
                 from /opt/libreoffice/include/tools/rc.hxx:23,
                 from /opt/libreoffice/include/vcl/outdev.hxx:26,
                 from /opt/libreoffice/include/vcl/window.hxx:25,
                 from /opt/libreoffice/include/vcl/vclevent.hxx:26,
                 from /opt/libreoffice/include/vcl/helper.hxx:26,
                 from /opt/libreoffice/include/vcl/printerinfomanager.hxx:27,
                 from /opt/libreoffice/vcl/null/printerinfomanager.cxx:20:
/usr/include/c++/5/bits/unique_ptr.h:169:7: note: candidate: std::unique_ptr<_Tp, _Dp>::unique_ptr(std::unique_ptr<_Tp, _Dp>::pointer) [with _Tp = psp::SystemQueueInfo; _Dp = std::default_delete<psp::SystemQueueInfo>; std::unique_ptr<_Tp, _Dp>::pointer = psp::SystemQueueInfo*]
       unique_ptr(pointer __p) noexcept
       ^
In file included from /usr/include/c++/5/memory:81:0,
                 from /opt/libreoffice/include/i18nlangtag/languagetag.hxx:19,
                 from /opt/libreoffice/include/tools/resmgr.hxx:23,
                 from /opt/libreoffice/include/tools/rc.hxx:23,
                 from /opt/libreoffice/include/vcl/outdev.hxx:26,
                 from /opt/libreoffice/include/vcl/window.hxx:25,
                 from /opt/libreoffice/include/vcl/vclevent.hxx:26,
                 from /opt/libreoffice/include/vcl/helper.hxx:26,
                 from /opt/libreoffice/include/vcl/printerinfomanager.hxx:27,
                 from /opt/libreoffice/vcl/null/printerinfomanager.cxx:20:
/usr/include/c++/5/bits/unique_ptr.h: In instantiation of â€˜void std::default_delete<_Tp>::operator()(_Tp*) const [with _Tp = psp::SystemQueueInfo]â€™:
/usr/include/c++/5/bits/unique_ptr.h:236:17:   required from â€˜std::unique_ptr<_Tp, _Dp>::~unique_ptr() [with _Tp = psp::SystemQueueInfo; _Dp = std::default_delete<psp::SystemQueueInfo>]â€™
/opt/libreoffice/vcl/null/printerinfomanager.cxx:47:33:   required from here
/usr/include/c++/5/bits/unique_ptr.h:74:22: error: invalid application of â€˜sizeofâ€™ to incomplete type â€˜psp::SystemQueueInfoâ€™
  static_assert(sizeof(_Tp)>0,
                      ^
make[1]: *** [/opt/libreoffice/workdir/CxxObject/vcl/null/printerinfomanager.o] Error 1
/opt/libreoffice/solenv/gbuild/LinkTarget.mk:189: recipe for target '/opt/libreoffice/workdir/CxxObject/vcl/null/printerinfomanager.o' failed
make[1]: *** Waiting for unfinished jobs....
make: *** [build] Error 2
Makefile:263: recipe for target 'build' failed

Looks like the compile has a print support error.

`/opt/libreoffice/vcl/null/printerinfomanager.cxx: In constructor â€˜psp::PrinterInfoManager::PrinterInfoManager(psp::PrinterInfoManager::Type)â€™:
/opt/libreoffice/vcl/null/printerinfomanager.cxx:47:33: error: call of overloaded â€˜unique_ptr(NULL)â€™ is ambiguous
m_aSystemDefaultPaper( "A4" )`

this option could be the culprit

--disable-cups

--- a/lib/default.cfg
+++ b/lib/default.cfg
@@ -17,7 +17,7 @@ lo_req_vol=11000
 lo_configure_opts=''
 lo_mini=true
 declare -r lo_mini_opts='--without-help --without-myspell-dicts --without-java --without-doxygen
---disable-cups --disable-dconf --disable-odk --without-junit --disable-dbus --disable-firebird-sdbc
+--disable-dconf --disable-odk --without-junit --disable-dbus --disable-firebird-sdbc
 --disable-postgresql-sdbc --disable-gltf --disable-extensions --disable-pdfimport --disable-neon
 --disable-lpsolve --disable-coinmp --disable-systray --disable-randr --disable-gstreamer-1-0
 --without-helppack-integration --disable-report-builder --disable-coinmp --disable-collada'

